deployable/module-template/variables.tf

```
@@ -0,0 +1,14 @@
+########################################################################################################################
+# Input variables
+########################################################################################################################
+
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud API Key"
+  sensitive   = true
+}
+
+variable "region" {
+  type        = string
+  description = "The IBM Cloud region"
```

```suggestion
  description = "The IBM Cloud region."
```

...where the resources are deployed?